### ISSUE TITLE
No hierarchichal relations on a dataset. Important errors in higher classification. 

### ISSUE THREAD

**Issue by DianRHR** [plazi/community#246]
Looking at dataset [Family-group names in Coleoptera (Insecta)](https://www.checklistbank.org/dataset/25656/classification?taxonKey=x35) noticed that it has no relations of parenthood among the names, so they all appear as individual records. 
![image](https://github.com/plazi/community/assets/124217901/4ae59407-626d-409a-9a18-625d54b15c4b)

Besides, the only two "trees" organized in the dataset are wrong: 
1. A "genus Coleoptera" in the Kingdom Plantae: 
![image](https://github.com/plazi/community/assets/124217901/8f12d544-7802-4c07-b5e0-b78dd7b7c8c4)

2. Order Blattodea in the kingdom Animalia, however, the article des not include any information regarding to the order Blattodea. 
![image](https://github.com/plazi/community/assets/124217901/36a57239-35b9-4a05-ad74-c52bd32fbb96)

It is not possible to use this dataset. Would be nice and useful if you do a thorough revision of the datasets already included in ChecklistBank.

**Comment by flsimoes** [comment]
@myrmoteras I guess this is something for Lyubo and/or Guido?

**Comment by camiplata** [comment]
@DianRHR is this issue still relevant for the extended release?  the latest Bouchard paper replaces this source?

**Comment by DianRHR** [comment]
@camiplata  most of the data in this dataset can be obtained from Bouchard et al 2024, however this dataset still lacks kinship relationships

**Comment by myrmoteras** [comment]
@camiplata and @DianRHR do you know which article/data set it refers? I can't build reconstruct the source of the article.

**Comment by flsimoes** [comment]
Closing this one as it has been stale for over an year

**Comment by DianRHR** [comment]
> [@camiplata](https://github.com/camiplata) and [@DianRHR](https://github.com/DianRHR) do you know which article/data set it refers? I can't build reconstruct the source of the article.

yes Donat its this one: https://zookeys.pensoft.net/articles.php?id=4001
But the main information included there is already in the Catalogue of Life.
